src/System.Windows.Forms.Primitives/src/System/LocalAppContextSwitches/LocalAppContextSwitches.cs

```diff
@@ -10,15 +10,32 @@ namespace System.Windows.Forms.Primitives
     // Borrowed from https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/System/LocalAppContextSwitches.Common.cs
     internal static partial class LocalAppContextSwitches
     {
+        // Switch names declared internal below are used in unit/integration tests. Refer to
+        // https://github.com/microsoft/winforms/blob/main/docs/AnchorLayoutChangesInNet80.md
+        // for more details on how to enable these switches in the application.
         private const string ScaleTopLevelFormMinMaxSizeForDpiSwitchName = "System.Windows.Forms.ScaleTopLevelFormMinMaxSizeForDpi";
+        internal const string AnchorLayoutV2SwitchName = "System.Windows.Forms.AnchorLayoutV2";
 
         private static int s_scaleTopLevelFormMinMaxSizeForDpi;
+        private static int s_AnchorLayoutV2;
+
         public static bool ScaleTopLevelFormMinMaxSizeForDpi
         {
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             get => GetCachedSwitchValue(ScaleTopLevelFormMinMaxSizeForDpiSwitchName, ref s_scaleTopLevelFormMinMaxSizeForDpi);
         }
 
+        /// <summary>
+        ///  Returns AnchorLayoutV2 switch value from runtimeconfig.json. Defaults to true if application is targeting .NET 8.0 and beyond.
```

Comment:
The comment has a lot of implementation details, which aren't enforcable. For example, nowhere here we read runtimeconfig.json. We also don't have any guarantees that it would be coming from there. 
Summary tag isn't suited for explaining the default values. Please refer to https://learn.microsoft.com/dotnet/csharp/language-reference/xmldoc/recommended-tags and https://review.learn.microsoft.com/en-au/help/contribute-ref/how-to-write-net-docs?branch=main.

```suggestion
        ///  Indicates whether AnchorLayoutV2 feature is enabled.
```

Reply:
@dreddy-work - you can use `<devdoc>` tag for comments that explain usage.